types/wp-parsely.php

```diff
@@ -0,0 +1,29 @@
+<?php
+/**
+ * This file contains types information which is needed by wp-parsely plugin.
```

Comment:
- Gives more info that helps in understanding of code.
- Helps IDE in autocompletion.

Reply:
I'm not sure about adding in a pattern like this and a new top-level folder into the broader mu-plugins repo. Could we drop this for now? Can put any helpful comments in the source files.